packages/did/src/Did.chain.ts

```diff
@@ -571,3 +571,9 @@ export function encodeDidSignature(
     [alg]: hexToU8a(signature.signature),
   }
 }
+
+export interface Web3NameOwner extends Struct {
+  owner: IDidIdentifier
+  claimedAt: BlockNumber
```

Comment:
I think in our case `BlockNumber` is a `u64`, while this one here seems to be `u32`. We should perhaps use `AnyNumber`, as above.

Reply:
fixed